gcn/notices/km3net/alert.schema.json

```diff
@@ -0,0 +1,85 @@
+{
+  "$id": "https://gcn.nasa.gov/schema/main/gcn/notices/km3net/alert.schema.json",
+  "$schema": "https://json-schema.org/draft/2020-12/schema",
+  "type": "object",
+  "unevaluatedProperties": false,
+  "title": "KM3NeT Alerts",
+  "description": "Candidate neutrino observation reported from the KM3NeT online analysis pipelines.",
+  "allOf": [
+    {
+      "$ref": "../neutrino/Alert.schema.json",
+      "description": "Neutrino alert schema. Alert info and most probable Localization (Time, Position, Error and Skymap if available)."
+    },
+    {
+      "$ref": "../core/AdditionalInfo.schema.json",
+      "description": "Nb. of pipeline triggering on this event, targeted FAR and details on selection."
+    }
+  ],
+  "properties": {
+    "$schema": true,
+    "pipeline": {
+      "enum": [
+        "exceptional_evt_arca",
+        "exceptional_evt_orca",
+        "multiplet_arca",
+        "multiplet_orca"
+      ],
+      "description": "Pipeline issuing this notice: either single, exceptional (high-energy), event or multiple coincident events, from KM3NeT/ARCA or KM3NeT/ORCA data."
+    },
+    "p_value": {
+      "type": "number",
+      "description": "Occurence probability of the alert reported."
+    },
+    "src_error_50": {
+      "type": "number",
+      "description": "50% CL uncertainty region described as a circle with a radius [deg]."
+    },
+    "event_details": {
+      "type": "array",
+      "items": { "$ref": "#/$defs/km3event" },
+      "description": "List and details of KM3NeT events involved in the alert."
+    },
+    "potential_counterpart": {},
+    "counterpart_info": {
+      "type": ["array", "string"],
+      "items": { "$ref": "#/$defs/knownsrc" },
+      "description": "No search / No match / (not exhaustive) Array of known astrophysical sources or transient present in the KM3NeT event RoI."
+    }
+  },
+  "$defs": {
+    "km3event": {
+      "type": "object",
+      "allOf": [
+        { "$ref": "../core/Event.schema.json" },
+        { "$ref": "../core/DateTime.schema.json" },
+        { "$ref": "../core/Localization.schema.json" }
+      ],
+      "properties": {
+        "prob_evt_topology": {
+          "enum": ["track", "starting track", "shower"],
+          "description": "Best reconstructed topology, possible to acknowledge if track starts inside the detector volume."
+        },
+        "cos_zenith": {
+          "type": "number",
+          "description": "Value of the cosine of the angle between the event direction and the vertical: 1 correspond to up-going event, -1 is down-going."
+        },
+        "proxy_energy": {
+          "type": "number",
+          "description": "Estimation of the energy of the lepton track/shower, resulting from the neutrino interaction [GeV]."
+        }
+      },
+      "description": "[Id, Date_Event, RA, Dec, Radius_Error, Topology, Zenith, reco_energy] of the KM3NeT events involved in the alert."
```

Comment:
This description does not need to list all of the properties, it could just say something like "The properties of the KM3NeT events that contributed to this alert."